roles/ovirt-ova-pack/files/pack_ova.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/python
+
+import os
+import io
+import tarfile
+import sys
+import time
+
+BLOCKSIZE = 512
+NUL = "\0"
+buf = bytearray(4096)
+
+def createTarInfo(name, size):
+    info = tarfile.TarInfo(name)
+    info.size = size
+    info.mtime = time.time()
+    return info
+
+def padToBlockSize(file, size):
+    remainder = size % BLOCKSIZE
+    if remainder > 0:
+        file.write(NUL * (BLOCKSIZE - remainder))
+
+ova_path = sys.argv[1]
+print "opening for write: %s" % ova_path
+ova_fd = os.open(ova_path, os.O_WRONLY)
+ova_file = io.FileIO(ova_fd, "w")
+
+ovf = sys.argv[2]
+print "writing ovf: %s" % ovf
+ovf_size = len(ovf.encode('utf-8'))
+tar_info = createTarInfo("ovf", ovf_size)
+ova_file.write(tar_info.tobuf())
+ova_file.write(ovf)
+padToBlockSize(ova_file, ovf_size)
+
+for disk_info in sys.argv[3:]:
+    idx = disk_info.index('::')
+    disk_path = disk_info[:idx]
+    disk_size = int(disk_info[idx+2:])
```

Comment:
Can you please comment what this does? Or what's the format of the `disk_info`?

Reply:
added a comment that describes this